languages/csharp/exercises/concept/constructors/Constructors.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+
+class RemoteControlCar
+{
+    // TODO: define the 'RemoteControlCar' classes' constructor
```

Comment:
But then it should be:

```suggestion
    // TODO: define the 'RemoteControlCar' class' constructor
```

I think :)

Reply:
Unfortunately not. EN-BR is my first language so there are some things I intuitively know, but I'm not great at explaining it! So I will leave this here instead, the usage here is as a posessive noun, similar to "Ross's car" : https://grammar.yourdictionary.com/punctuation/apostrophe-rules.html

Having read the linked page more thoroughly, I see that in cases where a singular noun ends in "s" it is acceptable to use the form you suggest, e.g. "Ross' ". However, having been a native EN-BR speaker for 30+ years I can say I have never noticed seeing this form in the wild.

Reply:
Do you have any idea if it would be more common for EN-US people? @robkeim?

Reply:
I only learned about a month ago that s's exists, and always was taught it didn't growing up.

It looks like there's event a divide among different style guides in the United States on the "right" way to do it:
https://www.prnewsonline.com/chicago-versus-AP-style

Reply:
Hmmm. We do want to use EN-US for our text. I think we use the single quote style elsewhere too. @iHiD @kytrinyx do you have any preference?

Reply:
I've [asked people to chip in with their thoughts](https://exercism-team.slack.com/archives/CR91YFNG3/p1588752272463400).

Reply:
How about an "alternate sentence" like the one proposed in this SE English post: [How do you write a classs constructor](https://english.stackexchange.com/questions/15609/how-do-you-write-a-classs-constructor)

Something like:
`TO DO: define the constructor for the 'RemoteControlCar' class`

Otherwise, it's allegedly: `class's constructor` ...which would be my first impulse.  But I also use oxford commas, so I am a bit of a freak. 😉

Reply:
1) I was taught both were correct.
2) I prefer CMoS generally because I am generally not writing for journalism.
3) [Purdue](https://owl.purdue.edu/owl/general_writing/punctuation/apostrophe_introduction.html) and [APA](https://blog.apastyle.org/apastyle/2013/06/forming-possessives-with-singular-names.html) favor the s's form (CMoS). [MLA](https://style.mla.org/apostrophes-three-ways/) favors the s' form (AP).

Of special note, [IEEE](https://www.ieee.org/content/dam/ieee-org/ieee/web/org/conferences/style_references_manual.pdf) is unequivocal on using an 's for possessive singular. And most of us do use IEEE standard _something_ in our code, already.

Finally, the Elements of Style is discussed [here](https://english.stackexchange.com/questions/62257/strunk-and-white-says-charless-is-correct-is-this-still-the-case) and the strong case is made that "Most speakers, however, do always say the /ɨz/ form for the possessive in all these, so they all do add a final s in writing". Yes, this is the case in spoken American English.

Reply:
Thanks for the detailed explanation! I quite like Bethany's suggestion to just rephrase and circumvent this entirely :)

Reply:
@wolf99 What do you think of the latest version?

Reply:
Looks good!